src/Symfony/Component/Workflow/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ CHANGELOG
 
  * Trigger `entered` event for subject entering in the Workflow for the first time
  * Added a context to `Workflow::apply()`. The `MethodMarkingStore` could be used to leverage this feature.
+ * Added dump styling of states and transitions through `metadata`,
+   configuration of places is:
+   `label` and `description` (only for PUML) (text), `bg_color` (`#RRGGBB` or CSS color name)
+   and configuration of transitions is:
+   `label` (text), `color` and `arrow_color` (`#RRGGBB` or CSS color name)
```

Comment:
Instead of describing in plain English, I suggest to write a PHP example on how to use the new feature, it should be more effective.

Reply:
Ok, we can take examples from the test file: https://github.com/symfony/symfony/pull/29538/files?file-filters%5B%5D=.md&file-filters%5B%5D=.php#diff-f7d48f660a821958cbe877f5eb738de5

And this part part should be moved to the documentation.

Reply:
let's do it!

Reply:
It's done.